vuu-ui/packages/vuu-data-react/src/data-editing/get-data-item-edit-control.tsx

```diff
@@ -53,6 +53,7 @@ export const getDataItemEditControl = ({
   if (dataDescriptor.editable === false) {
     return (
       <VuuInput
+        data-edit-control="data-edit-control"
```

Comment:
you could just use `data-edit-control` it will default to true, as we're just using it as a flag, that would work.